test/blackbox-tests/test-cases/pkg/alias-pkg-install.t

```diff
@@ -0,0 +1,48 @@
+This test verifies the @pkg-deps alias fetch and build the project dependencies
+without building the project itself.
+
+  $ . ./helpers.sh
+
+Create a fake library to install in the target project as a dependency:
+  $ mkdir foo
+  $ cd foo
+  $ cat > dune-project <<EOF
+  > (lang dune 3.16)
+  > (package (name foo))
+  > EOF
+  $ cat > foo.ml <<EOF
+  > let foo = "Hello, World!"
+  > EOF
+  $ cat > dune <<EOF
+  > (library
+  >  (public_name foo))
+  > EOF
+  $ cd ..
+  $ tar cf foo.tar foo
+  $ rm -rf foo
+
+Make an opam package for the library:
+  $ make_lockdir
+  $ cat > dune.lock/foo.pkg << EOF
+  > (version 0.0.1)
+  > (build
+  >  (run dune build -p %{pkg-self:name} @install))
+  > (source
+  >  (fetch (url file://$PWD/foo.tar)))
+  > EOF
+
+Create a project using the fake library as a dependency:
+  $ cat > dune-project << EOF
+  > (lang dune 3.16)
+  > (package
+  >  (name bar)
+  >  (allow_empty)
+  >  (depends foo))
+  > EOF
+
+The alias call builds the `foo` dependency but not the project itself. We
+verify we have the `foo` package but not the `bar.exe` in the build directory:
```

Comment:
This is confusing because there is no `bar.exe`; the project is empty.

How about creating a rule that will print something (maybe have to enable the debug print output flag) when it is run and show that is is not printed, but then build an executable which will then cause the rule to print? I think this is a bit clearer than exploring the contents of `_build`, as it uses the "public" API of Dune.

Reply:
This is a remaining of the past before I simplified the test. 

I don't know if it doesn't introduce more complexity than what we want?
Side note, but the debug package flag is enabled by default in `pkg/` tests :smile:

Reply:
Up to you, but I personally find it easier to read rules (since they are documented and and everyday tool in Dune) than to understand the meaning of what the existence or non-existence of files in `_build` means.